src/rendering/renderers/shared/extract/ExtractSystem.ts

```diff
@@ -230,7 +230,7 @@ export class ExtractSystem implements System
 
         const canvas = renderer.texture.generateCanvas(texture);
 
-        texture.destroy();
+        texture.destroy(true);
```

Comment:
Good catch. This is likely an issue with pixels() below too. Would you mind adding that as well?

Reply:
Sure! I missed it